docs/content/grafana/README.md

```diff
@@ -1,16 +1,35 @@
+</font>
+
+<font size="3">
+
+We use Grafana to visualize the metrics that we source from our [User Workload Monitoring][10] (click to learn more). These monitoring dashboards are useful for visualizing the metrics (such as Prometheus metrics) collected from the applications to track and analyze the system's overall health.
 
 The dashboards can be altered, saved and reused as per your specific monitoring needs.
 
-Dashboards can be represented by the `GrafanaDashboard` custom resource or by their raw `JSON` files. You can find example monitoring dashboards defined in separate `JSON` files for each of the Operate First managed applications in our [apps repo][2]. For more information on how the Grafana dashboards are defined, please refer to [this documentation][3].
+Dashboards can be represented by the `GrafanaDashboard` custom resource or by their raw `JSON` files. You can find example monitoring dashboards defined in separate `JSON` files for each of the Operate First managed applications in our apps repo found [here][2]. For more information on how the Grafana dashboards are defined, please refer to [this documentation][3] (includes information regarding dashboard properties, creation, UIDs, error handling, plugins, dashboard discovery, datasource inputs, config map references, and dashboard folder support).
+
+A dashboard can have a number of appearances and purposes. Here is a sample of what a simple dashboard might look like (this is the Operate First Grafana Dashboard landing page):
+
+![DashboardSample](img/dashboard-sample.png)
+This is the Operate First Grafana Dashboard landing page.
 
-## Get Access to Grafana
+## Get Access to the Grafana Instance
 
-Grafana can be accessed at: https://grafana.operate-first.cloud/
+In order to access the Grafana instance, you need to be in one of the **groups** found [here][11]. To add yourself to a group, add your GitHub username to its respective `group.yaml` file.
+
+***(Note: Group permissions vary, but if you are an intern/co-op you'll most likely be in the **SRE** group.)***
+
+Once your pull request is merged, the Grafana instance can be accessed at: https://grafana.operate-first.cloud/
+
+From there, click **Sign in with OAuth**. This will bring you to the OpenShift login page, and then (in most cases) you can select **operate-first** to access the landing dashboard.
+
+<font size = 4> _Don't have access?_ </font>
 
 You will need to be onboarded to access the UI, this can be self serviced, please follow the instructions [here][4] to do so. You will need to belong to an OCP `group` in order to gain access, current list of groups and their belonging users can be found [here][5]. If you do not belong to one, please create one, and then include this group in the `smaug` cluster overlay [here][6].
```

Comment:
there seems to be some duplication in the instructions here, can you revisit this and clarify. For example instructions to add to ocp groups are described twice.

Reply:
also instructions on _how to figure out what groups have access_ can also use clarification here I think